HLTriggerOffline/Btag/src/HLTBTagHarvestingAnalyzer.cc

```diff
@@ -163,6 +206,14 @@ bool HLTBTagHarvestingAnalyzer::GetNumDenumerators(DQMStore::IBooker& ibooker, D
 		delete cutg_num;
 		delete cutg_den;
 	}
+
+	if (type==3) //mistagrate HEP17 / HEM17 relation: fill "ptrnum" with HEP17 mistagrate and "ptrden" with HEM17 mistagrate
+	{
+		TH1* numH1 = numME->getTH1();
+		TH1* denH1 = denME->getTH1();
+		ptrden=(TH1*)denH1->Clone("denominator");
```

Comment:
One should *never* delete the pointers returned by `getTH1()` as they point to the objects owned by the DQM service.